user_guide.md

```diff
@@ -95,21 +100,53 @@ http://xx.yy.zz.ww:31942
 
 For some cloud deployments, the LoadBalancer service may take up to five minutes display an external IP address. Re-executing `kubectl get svc` repeatedly will eventually show the external IP field populated.
 
-Once you have an external IP, you can proceed to visit that in your browser. The hub by default is configured to take any username/password combination. After entering the username and password, you can start a single-notebook server,
-request any resources (memory/CPU/GPU), and then proceed to perform single node training.
+Once you have an external IP, you can proceed to visit that in your browser. You should see a sign in prompt.
```

Comment:
Do we need to change this? I don't think we should be opening an external IP by default. So I think we should tell people how to connect via kubectl until we have a recipe for connecting securely via https (#11).

Reply:
The security culture for k8s examples is pretty loose, however the documentation already mentions that by default this is totally insecure.

I do feel this is a continuation of a bad precedent.

A good solution would involve getting kube-cert-manager and external-dns configured, along with adding whitelists to the Service objects via ksonnet based on the user's IP, however I'm focusing on just getting things working for now.

I can switch everything to use port-forward if needed.

Reply:
I'd prefer for us to use port-forward and not promote insecure methods.

Reply:
Mind if I tackle the port-forward in another PR? It'll take me a bit to mess with ksonnet and put a toggle in there for loadbalancer vs port-forward (for those who want to use LB still)

Reply:
I think that's fine; since the code is already using LoadBalancer by default and this PR is just documenting current behavior.